integration/apocalypse/blockade.yaml

```diff
@@ -146,7 +146,7 @@ containers:
     #
 
     blockscout:
-        image: smartcontract/blockscout:apocalypse
+        image: puppeth/blockscout:latest
```

Comment:
I didn't know how to build that image!

Reply:
It should already be in our Dockerhub repo.  Are you unable to `docker pull` it?

Reply:
Yes, I managed to find it! Old comment :(